src/components/dev-hub/card-list.js

```diff
@@ -62,7 +61,7 @@ const renderArticle = article => (
 
 const renderVideo = video => (
     <VideoCard
-        key={video.title}
+        key={video.mediaType + video.title}
```

Comment:
I had to update these because titles were conflicting. For example, the team is uploading podcasts to youtube with the same title. This caused some crazy behavior. Mashing the type of media with title is safer.